packages/lib/services/plugins/api/JoplinViewsPanels.test.ts

```
@@ -0,0 +1,14 @@
+/* eslint-disable-next-line @typescript-eslint/triple-slash-reference */
+/// <reference types="jest" />
```

ESLint

[error] Expected space or tab after '//' in comment.

(spaced-comment)